microsetta_admin/server.py

```diff
@@ -964,11 +966,13 @@ def _post_bulk_scan():
                 obj = {}
                 obj["rack_id"] = rec[6]
                 if math.isnan(rec[3]):
-                    obj['location_col'] = 'None'
+                    error_msg = "Error: Empty column in row number" + rowCnt
+                    break
                 else:
                     obj["location_col"] = str(int(rec[3]))
                 if math.isnan(rec[4]):
```

Comment:
This isnan check needs to be removed, rec[4] will never be a number. Also, the error message strategy as written doesn't work - when it hits the isnan block, it throws an internal server error, meaning the user will never see the error.

Reply:
Just fixed the issue! As per the latest changes, if any sample has missing row or column in file used for bulk scan, the sample is avoided and the error is displayed accordingly. Please refer to the screenshot attached.
<img width="746" alt="Screen Shot 2023-05-12 at 9 34 21 AM" src="https://github.com/biocore/microsetta-admin/assets/110851289/11c9d0c1-d911-4b11-adeb-856ab0229897">

Reply:
Resolved the issued! As per the revised flow, samples without row or column numbers are ignored during bulk scan. Further, appropriate errors are displayed for such samples.

<img width="746" alt="Screen Shot 2023-05-12 at 9 34 21 AM" src="https://github.com/biocore/microsetta-admin/assets/110851289/7ad9f3e1-0224-425f-b24e-3f59bca2f74f">